vsintegration/src/FSharp.Editor/Commands/FsiCommandService.fs

```diff
@@ -36,15 +36,14 @@ type internal FsiCommandFilter(serviceProvider: System.IServiceProvider) =
 
     interface IOleCommandTarget with
         member x.Exec (pguidCmdGroup, nCmdId, nCmdexecopt, pvaIn, pvaOut) =
-            if pguidCmdGroup = VSConstants.VsStd11 then
-                if nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
-                elif nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteLineInInteractive then
-                    Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteLine null null
+            if pguidCmdGroup = VSConstants.VsStd11 && nCmdId = uint32 VSConstants.VSStd11CmdID.ExecuteSelectionInInteractive then
+                do Hooks.OnMLSend projectSystemPackage.Value FsiEditorSendAction.ExecuteSelection null null
```

Comment:
@JoshVarty just to give some context where do is used:

if you need some initialization code to run in constructor of types:

```fsharp
type MyType (myParameter) =
   do
      printfn "my parameter is: %s" myParameter
      // more code
   // rest of that type
```

also, can be useful to scope a disposable to a smaller scope than the function:

```fsharp
let myFunction () =
  let a = something ()
  do
      use willDisposeSoon = new SomethingDisposable()
      // ....
      // dispose called here
  doSomethingElse() |> ignore
  // some more code
```

In most other cases (I've encountered), it is mostly noise.